src/ChunkyImageLibTest/RectangleOperationTests.cs

```diff
@@ -13,31 +13,31 @@ public class RectangleOperationTests
 // to keep expected rectangles aligned
 #pragma warning disable format
     [Fact]
-    public void FindAffectedChunks_SmallStrokeOnly_FindsCorrectChunks()
+    public void FindAffectedArea_SmallStrokeOnly_FindsCorrectChunks()
```

Comment:
Well, some tests fail, but they were failing before this PR, I'm gonna let this pass